internal/kms/keyprotect.go

```diff
@@ -62,6 +63,20 @@ var _ = RegisterProvider(Provider{
 	Initializer: initKeyProtectKMS,
 })
 
+// RegisterProvider for kmsTypeKeyProtectMetadataOld is kept here for backward compatibility.
```

Comment:
open an issue for taking this one to remove it in 3.6

Reply:
https://github.com/ceph/ceph-csi/issues/2816